src/App.js

```diff
@@ -1,19 +1,161 @@
 import React, { Component } from 'react';
+import Row from './Row';
 
 export default class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      data: [],
+      sort: [],
+      checkSearch: false,
+      checkSort: false,
+      checkSortName: false,
+      checkSortAge: false,
+      item: {},
+      searched: []
+    }
+    this.handleClick = this.handleClick.bind(this);
+    this.handleSearch = this.handleSearch.bind(this);
+    this.handleNameSort = this.handleNameSort.bind(this);
+    this.handleAgeSort = this.handleAgeSort.bind(this);
+    this.handleClear = this.handleClear.bind(this);
+  }
 
   componentDidMount() {
     // Example of data fetching from server
     fetch('data.json')
       .then((httpResponse) => httpResponse.json())
-      .then((data) => console.log(data));
+      .then((data) => {
+        this.setState({data: data, store: data, item: data[0]})
+      });
+  }
+  handleClick(item) {
+    this.setState({item: item})
```

Comment:
Try to use object literal property value shorthand here like `this.setState({ item });`